analysis/06_additional_trigger_analysis.md

```diff
@@ -0,0 +1,991 @@
+---
+jupyter:
+  jupytext:
+    text_representation:
+      extension: .md
+      format_name: markdown
+      format_version: '1.3'
+      jupytext_version: 1.14.5
+  kernelspec:
+    display_name: bgdenv
+    language: python
+    name: python3
+---
+
+# Additional Analysis Requested by Hassan
+
+
+This notebook covers 3 analyses requested by Hassan for the trigger.
+
+- The FAR and POD calculated at Bahadurabad, Hardinge Bridge, and Goalando, in Excel file form. Note that for Goalando he wants us to compute it by (1) a point on the GloFAS raster, (2) by combining the discharge at Hardinge bridge with 1 day lag time with that at Bahadurabad with 2 day lag time
```

Comment:
Do you have pre-commit running? It should prevent long lines like this (max 80 char)